### DISCUSSION TITLE
null value in InvoiceItemCode

### DISCUSSION THREAD

**Discussion by EscapeToElSalvador** [btcpayserver/btcpayserver#5154] [Plugins & integrations]
When I export invoices, InvoiceItemCode returns a null value even though I have 'id' set for each product. Is there a different setting I have to declare in the product to get a code to show? Does the InvoiceItemCode show multiple codes if the invoice contains multiple products? If not, is there a way to export an invoice report with that data from the Greenfield API endpoint?

**Comment by pavlenex** [comment]
Hey, are you using Point of Sale products? If so that's not supported now, but stay tuned for next release where we will have way better reporting. https://github.com/btcpayserver/btcpayserver/pull/5155

**Reply by EscapeToElSalvador** [reply]
Ok, thank you, and yes. Just thought there might be a fix by throwing another command in the editor. For example, I have this as a product:

` {
    "description": "Chardonnay, 750ml bottle",
    "id": "Isla Negra Reserva Chardonnay, 750ml, Chile, 2021",
    "image": "linktoimg.jpg",
    "priceType": "Fixed",
    "price": "15",
    "title": "Wine - Chardonnay - Isla Negra Reserva",
    "inventory": 1,
    "disabled": false
  },`
  
  and I suspected that if I added something like:
  
  `
  "code": "sku12345"
  `
  
  Then it might show in the report...

**Comment by Joseedg** [comment]
0.00249629
